graphql-java-servlet/src/main/java/graphql/kickstart/servlet/OsgiSchemaBuilder.java

```diff
@@ -41,10 +41,12 @@
 import java.util.concurrent.ScheduledExecutorService;
 import java.util.concurrent.ScheduledFuture;
 import java.util.concurrent.TimeUnit;
+import java.util.stream.Stream;
+import lombok.Getter;
 import lombok.Setter;
 
 @Setter
-class OsgiSchemaBuilder {
+public class OsgiSchemaBuilder {
```

Comment:
Could you please create some unit tests for this builder since it has some login in it?